lib/internal/errors.js

```diff
@@ -154,12 +157,21 @@ E('ERR_UNKNOWN_BUILTIN_MODULE', (id) => `No such built-in module: ${id}`);
 E('ERR_UNKNOWN_SIGNAL', (signal) => `Unknown signal: ${signal}`);
 E('ERR_UNKNOWN_STDIN_TYPE', 'Unknown stdin file type');
 E('ERR_UNKNOWN_STREAM_TYPE', 'Unknown stream file type');
+E('ERR_RENEGOTIATE', 'Failed to renegotiate');
```

Comment:
The code should likely be a bit more specific... e.g. `ERR_TLS_RENEGOTIATION_FAILED`, etc